packages/p2p/src/components/buy-sell/currency-dropdown.jsx

```diff
@@ -41,8 +41,8 @@ const CurrencyDropdown = () => {
                     default_value={selected_local_currency}
                     list={local_currencies}
                     onSelect={value => {
-                        onLocalCurrencySelect(value);
                         setIsListVisible(false);
+                        onLocalCurrencySelect(value);
```

Comment:
is there any reason that you've changed the sequence here? or its unintentional?

Reply:
Yes @ali-hosseini-deriv it was a bug there, by changing the order of it the issue has gone